psamm/datasource/sbml.py

```diff
@@ -807,6 +826,50 @@ def write_model(self, file, model):
             reaction_tag.set(self._sbml_tag('reversible'), text_type(
                 equation.direction == Direction.Both).lower())
 
+            if eq_id not in flux_limits or flux_limits[eq_id][0] is None:
+                if equation.direction == Direction.Forward:
+                    lower_str = text_type(0)
+                else:
+                    lower_str = text_type(-model.get_default_flux_limit())
+            else:
+                lower_str = text_type(flux_limits[eq_id][0])
+            if eq_id not in flux_limits or flux_limits[eq_id][1] is None:
+                if equation.direction == Direction.Reverse:
+                    upper_str = text_type(0)
+                else:
+                    upper_str = text_type(model.get_default_flux_limit())
+            else:
+                upper_str = text_type(flux_limits[eq_id][1])
```

Comment:
We have code exactly like this in a few places now. Can you add a task on the trello board to refactor this?